code/modules/mob/living/simple_animal/abnormality/teth/MHz.dm

```diff
@@ -57,7 +59,8 @@
 		rose_available = TRUE
 		break
 
-	datum_reference.qliphoth_change(4)
+	addtimer(CALLBACK (src, .proc/Reset), reset_time)
```

Comment:
Pointless.
Simply add a timer to datum_reference for qliphoth change instead of creating new procs.

Reply:
You want a global timer using world time?

Reply:
No, I want this:
```suggestion
	addtimer(CALLBACK (datum_reference, /datum/abnormality/proc/qliphoth_change, 4), reset_time)
```

Reply:
Didn't know you could do that.